examples/How_to_handle_rate_limits.ipynb

```diff
@@ -43,9 +43,7 @@
     "\n",
     "### Requesting a rate limit increase\n",
     "\n",
-    "If you'd like your organization's rate limit increased, please fill out the following form:\n",
-    "\n",
-    "- [OpenAI Rate Limit Increase Request form](https://forms.gle/56ZrwXXoxAN1yt6i9)\n"
+    "If you'd like your organization's rate limit increased, please visit your [Limits settings page](https://platform.openai.com/account/limits) to see how you can increase your usage tier\n",
```

Comment:
`to see how you can increase your usage tier\n",`
the comma at the end broke the notebook.